pod/perlport.pod

```diff
@@ -2373,6 +2371,14 @@ have worked in 5.12, but supporting code has been removed for 5.14:
 
 =item Windows NT4
 
+=item Windows 2000
```

Comment:
This section is organized by version of removal and so there should be a new section added instead.